wiki/Tournaments/o!cCC/1/en.md

```diff
@@ -0,0 +1,234 @@
+  - [Siriusmo - Idiologie (Gens) \[Insane\]](https://osu.ppy.sh/beatmapsets/864954#fruits/1808752)
+  - [cosMo@BousouP feat. Hatsune Miku - Shinsekai (jonathanlfj) \[New World\]](https://osu.ppy.sh/beatmapsets/318557#fruits/709313)
+- HardRock
+  - [Dual & Dorothy starring Taneda Risa, Sakura Ayane - Oshiete (VINXIS) \[Remo Prototype\]](https://osu.ppy.sh/beatmapsets/451644#fruits/968814)
+  - [$44,000 - PISSCORD (Smoothie World) \[Mir's Insane\]](https://osu.ppy.sh/beatmapsets/691725#fruits/1475859)
+  - [ak+q - Rush Indigo (Alphabet) \[Somber\]](https://osu.ppy.sh/beatmapsets/986013#fruits/2062848)
+- DoubleTime
+  - [Insert Rupee - Facing Fears (OC ReMix) (Charles445) \[Insane\]](https://osu.ppy.sh/beatmapsets/36541#fruits/118054)
+  - [Nightcore - Ravers Fantasy (LoweH) \[Lizbeth's Fantasy\]](https://osu.ppy.sh/beatmapsets/11558#fruits/57382)
+  - [Yooh - salvation (ktgster) \[Another\]](https://osu.ppy.sh/beatmapsets/177444#fruits/477346)
+- Tiebreaker
+  - **[goreshit - burn this moment into the retina of my eye (grumd) \[extra\]](https://osu.ppy.sh/beatmapsets/359890#fruits/791274)**
+
+### Round of 32
+
+**[Download the mappack here! (83 MB)](https://mega.nz/#!8kVg3QrC!gtqqHtYkvbXksiAuhJJOv_8eKcSqc3tbApL-EzaqvVc)**
+
+- NoMod
+  - [Unlucky Morpheus - Phantom Blood (IsomirDiAngelo) \[Atalanta's Extra\]](https://osu.ppy.sh/beatmapsets/885158#fruits/2153771)
+  - [Coda - BLOODY STREAM (Shiguma) \[OVERDRIVE\]](https://osu.ppy.sh/beatmapsets/674882#fruits/1480932)
+  - [twenty one pilots - Levitate (Mao) \[Vulture\]](https://osu.ppy.sh/beatmapsets/1026205#fruits/2146132)
+  - [Akira Complex - Ether Strike (Mun) \[Mir's Aethereal Expert\]](https://osu.ppy.sh/beatmapsets/945019#fruits/1973467)
+- Hidden
+  - [Rammstein - Feuer frei! (Frey) \[Fire at will! BANG! BANG!!\]](https://osu.ppy.sh/beatmapsets/887102#fruits/1854467)
+  - [Gesu no Kiwami Otome. - DARUMASAN (celerih) \[dsco's EXTREME\]](https://osu.ppy.sh/beatmapsets/613613#fruits/1302779)
+  - [Grand Thaw - Aventyr (kwk) \[Adventure\]](https://osu.ppy.sh/beatmapsets/556922#fruits/1178554)
+- HardRock
+  - [Morimori Atsushi+Ujico\* - Vieille Ville (cheesiest) \[diff\]](https://osu.ppy.sh/beatmapsets/484162#fruits/1032837)
+  - [Camellia - Diastrophism (Regou) \[Insane\]](https://osu.ppy.sh/beatmapsets/923095#fruits/1932099)
+  - [Glutamine - Babylon (CSY the corrupt) \[Corruption\]](https://osu.ppy.sh/beatmapsets/35992#fruits/145118)
+- DoubleTime
+  - [Katy Perry - E.T. (La Cataline) \[Futuristic\]](https://osu.ppy.sh/beatmapsets/36077#fruits/116708)
+  - [Dj Verstarker - Jump Jump (2008) (SeymourLyon) \[Jump like CRAZY!!!\]](https://osu.ppy.sh/beatmapsets/6004#fruits/28152)
+  - [Sharlo - Shinkai Shoujo (yf\_bmp) \[Insane\]](https://osu.ppy.sh/beatmapsets/221768#fruits/518731)
+- Tiebreaker
+  - **[Feryquitous - Yoi no Tamoto, Yoru to Futari (Ametrin) \[Monogatari\]](https://osu.ppy.sh/beatmapsets/891830#fruits/1864425)**
+
+### Qualifiers Stage
```

Comment:
sentence case headers (not sure if "Qualifiers Stage" is a proper noun or not, depends how the tournament info formats it)

same w/ any headers below this one

Reply:
well now it's all over the place lol

"Registration Phase" vs "Qualifiers stage" ... sentence case everything then?
"Qualifiers stage" vs "Qualifier Stage Details" (should be Qualifier stage details)

and the other headers i mention are still the same

Reply:
Something like Grand finals too?

Reply:
depends if the rest of the tournament considers that a proper noun, i dont know

Reply:
> same w/ any headers below this one

are u gonna fix this 😢